### ISSUE TITLE
version `GLIBCXX_3.4.32' not found

### ISSUE THREAD

**Issue by Autumnlight02** [LostRuins/koboldcpp#453]
# Prerequisites

Please answer the following questions for yourself before submitting an issue.

- [ x] I am running the latest code. Development is very rapid so there are no tagged versions as of now.
- [x ] I carefully followed the [README.md](https://github.com/ggerganov/llama.cpp/blob/master/README.md).
- [ x] I [searched using keywords relevant to my issue](https://docs.github.com/en/issues/tracking-your-work-with-issues/filtering-and-searching-issues-and-pull-requests) to make sure that I am creating a new issue that is not already open (or closed).
- [x ] I reviewed the [Discussions](https://github.com/ggerganov/llama.cpp/discussions), and have a new bug or useful enhancement to share.

# Expected Behavior

The software should run

# Current Behavior

When i try to load a Model I get: 

`Welcome to KoboldCpp - Version 1.44.2
For command line arguments, please refer to --help
***
Attempting to use OpenBLAS library for faster prompt ingestion. A compatible libopenblas will be required.
Initializing dynamic library: koboldcpp_openblas.so
Traceback (most recent call last):
  File "/home/autumnlight/Documents/ai/koboldcpp/./koboldcpp.py", line 1914, in <module>
    main(parser.parse_args(),start_server=True)
  File "/home/autumnlight/Documents/ai/koboldcpp/./koboldcpp.py", line 1781, in main
    init_library() # Note: if blas does not exist and is enabled, program will crash.
    ^^^^^^^^^^^^^^
  File "/home/autumnlight/Documents/ai/koboldcpp/./koboldcpp.py", line 184, in init_library
    handle = ctypes.CDLL(os.path.join(dir_path, libname))
             ^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
  File "/home/autumnlight/.distrobox/archo/miniconda3/lib/python3.11/ctypes/__init__.py", line 376, in __init__
    self._handle = _dlopen(self._name, mode)
                   ^^^^^^^^^^^^^^^^^^^^^^^^^
OSError: /home/autumnlight/.distrobox/archo/miniconda3/bin/../lib/libstdc++.so.6: version `GLIBCXX_3.4.32' not found (required by /home/autumnlight/Documents/ai/koboldcpp/koboldcpp_openblas.so)
`

# Environment and Context

Inside of an arch distrobox, strings /usr/lib/libstdc++.so.6 | grep GLIBCXX =>

...
GLIBCXX_3.4.27
GLIBCXX_3.4.28
GLIBCXX_3.4.29
GLIBCXX_3.4.30
GLIBCXX_3.4.31
GLIBCXX_3.4.32
...

* Physical (or virtual) hardware you are using, e.g. for Linux:

`$ lscpu`

* Operating System, e.g. for Linux:

Linux archo.fedora 6.3.12-100.fc37.x86_64 #1 SMP PREEMPT_DYNAMIC Wed Jul  5 20:09:58 UTC 2023 x86_64 GNU/Linux
Basically host is fedora
Inside of an isolated arch distrobox.

* SDK version, e.g. for Linux:

```
$ python3 --version
Python 3.11.4
$ make --version
GNU Make 4.4.1
$ g++ --version
g++ (GCC) 13.2.1 20230801
```

# Failure Information (for bugs)

BUG

# Steps to Reproduce

Please provide detailed steps for reproducing the issue. We are not sitting in front of your screen, so the more detail the better.

1. Cloning the git
2. Executing make
3. Running the software
4 loading a model
5 ded

**Comment by LostRuins** [comment]
Following up on discord with Henky.

**Comment by morpheus2448** [comment]
I had this problem today.

For me the issue was the order of my paths in the PATH env picking up libstdc++.so.6 from an anaconda install.

I solved this error by setting the PATH env before running koboldcpp.py:

> cd koboldcpp
> PATH=/usr/bin ./koboldcpp.py --model models/dolphin-2.5-mixtral-8x7b.Q5_K_M.gguf --contextsize 8192 --launch

Running on Fedora 39

**Comment by johnwas** [comment]
It seems that the conflict was caused by conda runtime. I solved it as follows:

conda install -c conda-forge libstdcxx-ng
